fdbclient/include/fdbclient/Atomic.h

```diff
@@ -299,9 +302,7 @@ inline void transformVersionstampMutation(MutationRef& mutation,
                                           Version version,
                                           uint16_t transactionNumber) {
 	if ((mutation.*param).size() >= 4) {
-		int32_t pos;
-		memcpy(&pos, (mutation.*param).end() - sizeof(int32_t), sizeof(int32_t));
-		pos = littleEndian32(pos);
+		auto pos = parseGetVersionStampKetOffset(mutation.*param);
```

Comment:
This function could technically also be using a value, so I would suggest naming the function more generically, like `parseVersionstampOffset`. Note also that versionstamp is one word, so the 's' shouldn't be capitalized.